src/components/timeline/TrackMemoryGraph.js

```diff
@@ -149,10 +151,7 @@ class TrackMemoryCanvas extends React.PureComponent<CanvasProps> {
       // After doing the stroke, continue the path to complete the fill to the bottom
       // of the canvas.
       ctx.lineTo(x + interval, deviceHeight);
-      ctx.lineTo(
-        (deviceWidth * (samples.time[0] - rangeStart)) / rangeLength + interval,
-        deviceHeight
-      );
+      ctx.lineTo(firstX, deviceHeight);
```

Comment:
I understand the difference is the `+ interval` at the end, which seemed wrong indeed for the start.
But I don't understand why this fixes this specific issue where the problem seems to be with the `y` parameter (`deviceHeight` here), that doesn't change.
Could you please explain what the problem was and why your fix fixes it?

Adding a test that reproduces the issue would help understand this too.

Reply:
I added clarifying comments here with the drawing strategy, but didn't add a test as the snapshot reflected the change, and this is such a small fix.

Reply:
Also, I didn't try to understand that original line, but the issue was that the final shape drawing wasn't aligned with the first X value. It seemed easiest to just remember the value compared to re-computing it.